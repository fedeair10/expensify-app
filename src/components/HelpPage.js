import React from 'react';

const HelpPage = () => (
	<div>This is from by help component</div>
);

export default HelpPage;