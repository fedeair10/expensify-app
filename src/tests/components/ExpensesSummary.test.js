import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary with one expense', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={123} />);
	expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={3} expensesTotal={114195} />);
	expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with empty message', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={0} expensesTotal={0} />);
	expect(wrapper).toMatchSnapshot();
});