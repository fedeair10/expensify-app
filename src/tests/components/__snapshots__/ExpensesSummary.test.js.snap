// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with empty message 1`] = `
<div>
  <h1>
    No expesens selected
  </h1>
</div>
`;

exports[`should render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $1,141.95
  </h1>
</div>
`;

exports[`should render ExpensesSummary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $1.23
  </h1>
</div>
`;
